### PR TITLE
Return drug identifiers in DDI response

### PR DESCRIPTION
**Pull request by shawa**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -133,14 +133,12 @@ Automated tests confirm that this feature works as intended. You can run the tes
 specific to this feature by running ~docker run -t -e "MIX_ENV=test" tomtoothfairies/panacea mix test --only identify_ddis~.
 
 *** DINTO - Asclepius ⚕
-  Asclepius provides an endpoint for querying DINTO. It supports querying for
-  all drugs listed within its given Ontology, as well as finding all, or
-  specific drug-drug interactions.
+  Asclepius provides an endpoint for querying DINTO. It supports querying for all drugs listed within its given Ontology, as well as finding all, or specific drug-drug interactions.
```
Comment (houli) [inline README.org]: This hunk should be reverted

File: panacea/lib/panacea/asclepius/remote/mock.ex [hunk 2]
```diff
@@ -1,16 +1,21 @@
 defmodule Panacea.Asclepius.Remote.Mock do
   @behaviour Panacea.Asclepius.Remote
 
+  def ping(), do: {:ok, nil}
```
Comment (c-brenn) [inline panacea/lib/panacea/asclepius/remote/mock.ex]: ping should be gone

File: README.org [hunk 3]
```diff
@@ -133,14 +133,12 @@ Automated tests confirm that this feature works as intended. You can run the tes
...
 **** Setup
 ***** DINTO Ontology SPARQL Endpoint
-This application acts as an adaptor to an instance of Apache Fuseki which can
-serve SPARQL queries on a DINTO ontology. Fuseki must be running before any
-queries can be served.
+This application acts as an adaptor to an instance of Apache Fuseki which can serve SPARQL queries on a DINTO ontology. Fuseki must be running before any queries can be served.
```
Comment (houli) [inline README.org]: This hunk should be reverted

File: panacea/lib/panacea/asclepius/remote/mock.ex [hunk 4]
```diff
@@ -1,16 +1,21 @@
...
   def ddis(_drugs) do
-    {:ok, [
-      %{
-        "label" => "abacavir/ritonavir DDI",
-        "uri" => "http://purl.obolibrary.org/obo/DINTO_11043"
-      },
+    [
```
Comment (c-brenn) [inline panacea/lib/panacea/asclepius/remote/mock.ex]: this should be `{:ok, [ddis]}` instead of `[ddis]`

**Review by c-brenn — CHANGES_REQUESTED** [review]

File: asclepius/asclepius/main.py [hunk 5]
```diff
@@ -1,11 +1,15 @@
...
 
-from flask import jsonify, request
-
+from flask import Flask, jsonify, request
 from app import app
+
 import dinto
 
+
+app = Flask(__name__)
```
Comment (houli) [inline asclepius/asclepius/main.py]: this shouldn't be here, app lives in app.py

File: asclepius/asclepius/main.py [hunk 6]
```diff
@@ -1,11 +1,15 @@
 import logging
 logging.basicConfig(level=logging.INFO)
 
-from flask import jsonify, request
-
+from flask import Flask, jsonify, request
```
Comment (houli) [inline asclepius/asclepius/main.py]: revert this, no longer need Flask in this module

**Comment by houli** [comment]
`pml_controller_test.exs` also needs to be updated to test that the response has drug_a and drug_b

**Review by c-brenn — APPROVED** [review]

**Pull request merged**